ui/src/app/settings/components/repos-list/repos-list.tsx

```diff
@@ -284,84 +294,188 @@ export class ReposList extends React.Component<
+                                                items={projectItems}
+                                                anchor={() => (
+                                                    <>
+                                                        <a style={{whiteSpace: 'nowrap'}}>
+                                                            Project: {this.state.projectProperty} <i className='fa fa-caret-down' />
+                                                        </a>
+                                                        &nbsp;
+                                                    </>
+                                                )}
+                                                qeId='project-menu'
+                                            />
+                                        );
+                                    }}
+                                </DataLoader>
+                                <DropDownMenu
+                                    items={[
+                                        {
+                                            title: 'all',
+                                            action: () => this.setState({statusProperty: 'all'})
+                                        },
+                                        {
+                                            title: 'Successful',
+                                            action: () => this.setState({statusProperty: 'Successful'})
+                                        },
+                                        {
+                                            title: 'Failed',
+                                            action: () => this.setState({statusProperty: 'Failed'})
+                                        },
+                                        {
+                                            title: 'Unknown',
+                                            action: () => this.setState({statusProperty: 'Unknown'})
+                                        }
+                                    ]}
+                                    anchor={() => (
+                                        <>
+                                            <a style={{whiteSpace: 'nowrap'}}>
+                                                Status: {this.state.statusProperty} <i className='fa fa-caret-down' />
+                                            </a>
+                                            &nbsp;
+                                        </>
+                                    )}
+                                    qeId='status-menu'
+                                />
+                            </div>
+                            <div className='search-bar' style={{display: 'flex', alignItems: 'flex-end', width: '100%'}}></div>
+                            <input type='text' className='argo-field' placeholder='Search Name' value={this.state.name} onChange={e => this.setState({name: e.target.value})} />
```

Comment:
This search field initially doesn't seem to be clickable, until you actually click it.  eg. it looks more like a label

Reply:
Let me add some color.